.github/workflows/publish-to-npm.yml

```diff
@@ -108,7 +108,7 @@ jobs:
     needs: promote-to-latest
     runs-on: macos-latest
     env:
-      GH_TOKEN: ${{ github.token }}
+      GH_TOKEN: ${{ secrets.SVC_CLI_BOT_GITHUB_TOKEN }}
```

Comment:
Pretty sure we actually wanted to be using the CLI Bot's PAT, so that the workflows about validating PRs will automatically kick off.